demos/bubblewrap/actors/bubble.py

```diff
@@ -1,10 +1,13 @@
 import numpy as np
 from queue import Empty
 import os
-
 import improv.store as store
-
-from bubblewrap import Bubblewrap
+from scipy import io as sio
```

Comment:
This is the idea, but we should have this for _every_ library that is not part of our dependencies in `pyproject.toml`.  That includes `mat73` and `pyqtgraph`.